Description: this patch does two things. First, it removes every entry that is
not used, as the only entry point available is the command 'py' itself. Then it
updates 'pycompleter' path to use the internal executable directly.
Forwarded: not-needed
Last-Update: 2016-05-20
Author: Tiago Ilieve <tiago.myhro@gmail.com>

--- a/pythonpy/completion/pycompletion.sh
+++ b/pythonpy/completion/pycompletion.sh
@@ -1,59 +1,6 @@
 _py()
 {
-    COMPREPLY=($(pycompleter "${COMP_WORDS[@]}" 2>/dev/null | sed 's/.*1034h//'))
-    if [[ ${COMPREPLY[0]} == '_longopt' ]]; then
-        COMPREPLY=()
-        _longopt 2>/dev/null
-    fi
-}
-
-_py2()
-{
-    COMPREPLY=($(pycompleter2 "${COMP_WORDS[@]}" 2>/dev/null | sed 's/.*1034h//'))
-    if [[ ${COMPREPLY[0]} == '_longopt' ]]; then
-        COMPREPLY=()
-        _longopt 2>/dev/null
-    fi
-}
-
-_py2.6()
-{
-    COMPREPLY=($(pycompleter2.6 "${COMP_WORDS[@]}" 2>/dev/null | sed 's/.*1034h//'))
-    if [[ ${COMPREPLY[0]} == '_longopt' ]]; then
-        COMPREPLY=()
-        _longopt 2>/dev/null
-    fi
-}
-
-_py2.7()
-{
-    COMPREPLY=($(pycompleter2.7 "${COMP_WORDS[@]}" 2>/dev/null | sed 's/.*1034h//'))
-    if [[ ${COMPREPLY[0]} == '_longopt' ]]; then
-        COMPREPLY=()
-        _longopt 2>/dev/null
-    fi
-}
-
-_py3()
-{
-    COMPREPLY=($(pycompleter3 "${COMP_WORDS[@]}" 2>/dev/null | sed 's/.*1034h//'))
-    if [[ ${COMPREPLY[0]} == '_longopt' ]]; then
-        COMPREPLY=()
-        _longopt 2>/dev/null
-    fi
-}
-
-_py3.3()
-{
-    COMPREPLY=($(pycompleter3.3 "${COMP_WORDS[@]}" 2>/dev/null | sed 's/.*1034h//'))
-    if [[ ${COMPREPLY[0]} == '_longopt' ]]; then
-        COMPREPLY=()
-        _longopt 2>/dev/null
-    fi
-}
-_py3.4()
-{
-    COMPREPLY=($(pycompleter3.4 "${COMP_WORDS[@]}" 2>/dev/null | sed 's/.*1034h//'))
+    COMPREPLY=($(/usr/share/pythonpy/pythonpy/pycompleter.py "${COMP_WORDS[@]}" 2>/dev/null | sed 's/.*1034h//'))
     if [[ ${COMPREPLY[0]} == '_longopt' ]]; then
         COMPREPLY=()
         _longopt 2>/dev/null
@@ -62,9 +9,3 @@
 
 
 complete -F _py -o nospace py
-complete -F _py2 -o nospace py2
-complete -F _py2.6 -o nospace py2.6
-complete -F _py2.7 -o nospace py2.7
-complete -F _py3 -o nospace py3
-complete -F _py3.3 -o nospace py3.3
-complete -F _py3.4 -o nospace py3.4
